integration/src/main/java/scripts/testscripts/PrivateControlledAiNotebookInstanceLifecycle.java

```diff
@@ -120,12 +97,11 @@ protected void doUserJourney(TestUserSpecification testUser, WorkspaceApi worksp
             resource.getAttributes().getLocation(), resource.getAttributes().getInstanceId());
     AIPlatformNotebooks userNotebooks = ClientTestUtils.getAIPlatformNotebooksClient(resourceUser);
 
-    Instance instance = userNotebooks.projects().locations().instances().get(instanceName)
-        .execute();
-
     // TODO(PF-765): Assert that the other user does not have proxy access once we don't have
     // project level service account permissions.
-    assertUserHasProxyAccess(resourceUser, instance, resource.getAttributes().getProjectId());
+    assertTrue(ResourceMaker.userHasProxyAccess(
```

Comment:
nit: `ResourceMaker` is now doing more than just making resources, so maybe it needs a new name or a new class split out.

Reply:
agreed. split this into `ResourceMaker` and `ResourceModifier`